util/mbed/mbed-copy-osx.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```

Comment:
We can probably re-use a lot of the code in the linux mbed-copy script.  Maybe we can just have it be one script and just look at different paths or query the system a little differently based on the platform.